maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/AbstractGeneratorMojo.java

```diff
@@ -185,8 +185,8 @@
     private List<String> expectedProvidedScopeGroupIds = Collections.singletonList( "org.apache.maven" );
 
     /**
-     * List of {@code G:A} strings of artifact coordinates that are to be excluded from "expected provided scope" check.
-     * Default value: {@code ["org.apache.maven:maven-archiver", "org.apache.maven:maven-jxr"]}.
+     * List of {@code groupID:artifactID} strings of artifact coordinates that are to be excluded from "expected
```

Comment:
Well, the field above days `groupId`, not `groupID` and the former is the term we use everywhere.

Reply:
Fixed